src/machines/sketchSolve/sketchSolveMode.ts

```diff
@@ -1,37 +1,38 @@
-import { assign, assertEvent, createMachine, sendTo, setup } from 'xstate'
+import { assertEvent, assign, createMachine, sendTo, setup } from 'xstate'
 import type { ActorRefFrom } from 'xstate'
 import { modelingMachineDefaultContext } from '@src/machines/modelingSharedContext'
 import type {
   ModelingMachineContext,
   SetSelections,
-  MouseState,
-  SegmentOverlayPayload,
 } from '@src/machines/modelingSharedTypes'
-import type { PathToNode } from '@src/lang/wasm'
-import { machine as centerRectToolMachine } from '@src/machines/centerRectTool'
-import { machine as dimensionToolMachine } from '@src/machines/dimensionTool'
+import { machine as centerRectTool } from '@src/machines/sketchSolve/tools/centerRectTool'
+import { machine as dimensionTool } from '@src/machines/sketchSolve/tools/dimensionTool'
+import { machine as pointTool } from '@src/machines/sketchSolve/tools/pointTool'
 
-type EquipTool = 'dimension' | 'center rectangle'
+const equipTools = {
+  centerRectTool,
+  dimensionTool,
+  pointTool,
+} as const
```

Comment:
```suggestion
const equipTools = Object.freeze({
  centerRectTool,
  dimensionTool,
  pointTool,
})
```

[We've discussed making a lint for this in Slack](https://kittycadworkspace.slack.com/archives/C0921NRC35L/p1758207105875709?thread_ts=1758205041.718059&cid=C0921NRC35L), but I still need to make an issue for that. With objects, Object.freeze gives you the same definite typing _and_ actually prevents runtime edits.

Reply:
Yeah I've confirmed this works has no negative impacts, I'm gonna commit it.